cmd/controller.go

```diff
@@ -131,3 +133,13 @@ func convertAnnotationToMap(s []string) (map[string]string, error) {
 	}
 	return m, nil
 }
+
+func buildKubeClientConfig(masterURL string, kubeConfigPath string, timeout time.Duration) (*rest.Config, error) {
```

Comment:
I'm not sure but would placing this inside the [`kubernetes`](https://github.com/hellofresh/kangal/tree/master/pkg/kubernetes) package not be cleaner?

Reply:
Done via 8aad2bc